probatus/feature_elimination/feature_elimination.py

```diff
@@ -408,7 +436,7 @@ def fit(self, X, y, column_names=None):
                       f'+/- {self.report_df.loc[round_number]["train_metric_std"]}, CV Validation '
                       f'{self.report_df.loc[round_number]["val_metric_mean"]} '
                       f'+/- {self.report_df.loc[round_number]["val_metric_std"]}. \n'
-                      f'Num of features left: {len(remaining_features)}. '
+                      f'Features left: {len(remaining_features)}. '
```

Comment:
The remaining features should also contain the columns_to_keep. This is for reporting which columns are still in the dataset.

Reply:
Yes, the columns to keep are in the remaining column.